specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2019-08-01-preview/Clusters.json

```diff
@@ -470,6 +469,40 @@
       },
       "description": "The list clusters operation response."
     },
+    "keyVaultProperties": {
+      "properties": {
+        "keyVaultUri": {
+          "description": "The Key Vault uri which holds they key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyName": {
+          "description": "The name of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        },
+        "keyVersion": {
+          "description": "The version of the key associated with the Log Analytics cluster.",
+          "type": "string"
+        }
+      }
+    },
+    "sku": {
+      "properties": {
+        "capacityReservationLevel": {
+          "description": "The value of the Capacity Reservation in GB per day",
+          "enum": [
+            "CapacityReservation"
+          ],
+          "x-ms-enum": {
+            "name": "SkuNameEnum",
+            "modelAsString": true
+          }
```

Comment:
The enum and the description don't make sense together here.  Also, "sku" is a standard contract.  Can this be placed in the "capacity" field instead?

Reply:
Thanks for the comment. This Enum description was misplaced and should have been under name.
Regarding the name capacity, i am aware of the documentation and agree in general. The problem is that we already have this property named capacityReservationLevel under the workspace resource so we prefer not to confuse the customer with different property names for something that has the same logic.

Reply:
Sku is a standard contract.  The workspaces don't have this property as part of their "sku" object.  Its exposed there elsewhere in serviceTiers.  If it is part of the sku contract it needs to be named "capacity"

Reply:
Fixed